spec/models/data_file_spec.rb

```diff
@@ -245,48 +245,75 @@
     it { is_expected.to callback(:set_current_file_version_attributes).before(:save) }
   end
 
-  describe 'elasticsearch' do
-    it { expect(described_class).to include(Elasticsearch::Model) }
-
-    # TODO, when we move to asynchronous indexing, remove this and replace with
-    # a test to ensure that jobs are created on create, update, delete
-    it { expect(described_class).to include(Elasticsearch::Model::Callbacks) }
-
-    it { is_expected.to respond_to 'as_indexed_json' }
-    it { expect(subject.as_indexed_json).to eq(DataFileSearchDocumentSerializer.new(subject).as_json) }
+  describe 'creator' do
+    let(:creator) { FactoryGirl.create(:user) }
+    subject {
+      Audited.audit_class.as_user(creator) do
+        FactoryGirl.create(:data_file)
+      end
+    }
+    it { is_expected.to respond_to :creator }
+    it {
+      expect(subject.current_file_version).not_to be_nil
+      expect(subject.current_file_version.audits).not_to be_empty
+      expect(subject.current_file_version.audits.find_by(action: 'create')).not_to be_nil
```

Comment:
These 3 expectations (lines 257-9) could be broken out into their own contexts, which will ultimately test the guard clause in your creator method.